job-templates/kubernetes_1903_master.json

```diff
@@ -23,7 +23,7 @@
       "count": 1,
       "dnsPrefix": "",
       "vmSize": "Standard_D2_v3",
-      "distro": "ubuntu",
+      "distro": "aks-ubuntu-18.04",
```

Comment:
Is possible to set the VHD imageVersion explicitly?  This way we know the version that is deployed is consistent across tests?

Reply:
AFAIK aks-engine API model doesn't allow users to override the Linux image version